vowpalwabbit/ccb_label.h

```diff
@@ -40,5 +42,13 @@ struct label
   float weight;
 };
 
+struct predict_data
+{
+  example_type type;
+  v_array<uint32_t> explicit_included_actions;
```

Comment:
are we leaving this in even if its still not used?

Reply:
I think its fine to leave this here for now. There should be another PR later to remove the variables from label and switch the usage over to this one